### ISSUE TITLE
Install docs refer to incorrect package name

### ISSUE THREAD

**Issue by asachs01** [unpoller/unpoller#408]
# Expected behavior
When running the command from https://unpoller.com/docs/install/linux, I should be able to install Unpoller with no errors:
```
curl -s https://golift.io/repo.sh | sudo bash -s - unpoller
```

# Current behavior
When using the installer script from https://unpoller.com/docs/install/linux, the command references `unpoller` as a package name, which results in a `package not found` error:

<details>
  <summary>Error when installing unpoller</summary>

```
# curl -s https://golift.io/repo.sh | sudo bash -s - unpoller

OK
Hit:1 http://us.archive.ubuntu.com/ubuntu focal InRelease
Get:2 http://us.archive.ubuntu.com/ubuntu focal-updates InRelease [114 kB]
Hit:3 https://packages.grafana.com/oss/deb stable InRelease
Hit:4 https://repos.influxdata.com/ubuntu focal InRelease
Get:5 http://us.archive.ubuntu.com/ubuntu focal-backports InRelease [108 kB]
Get:7 http://us.archive.ubuntu.com/ubuntu focal-security InRelease [114 kB]
Get:9 https://pkgs.tailscale.com/stable/ubuntu focal InRelease
Get:10 http://us.archive.ubuntu.com/ubuntu focal-updates/main amd64 Packages [1,712 kB]
Get:6 https://packagecloud.io/golift/pkgs/ubuntu focal InRelease [24.5 kB]
Get:12 http://us.archive.ubuntu.com/ubuntu focal-updates/universe amd64 Packages [918 kB]
Hit:8 https://packages.graylog2.org/repo/debian sidecar-stable InRelease
Hit:11 https://packagecloud.io/sensu/stable/ubuntu focal InRelease
Get:13 https://packagecloud.io/golift/pkgs/ubuntu focal/main amd64 Packages [3,661 B]
Fetched 3,000 kB in 2s (1,580 kB/s)
Reading package lists... Done
Building dependency tree
Reading state information... Done
33 packages can be upgraded. Run 'apt list --upgradable' to see them.
Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Unable to locate package unpoller
```

</details>

Instead, the command I'm having to use is referencing `unifi-poller` as the package name:

<details>
  <summary>Success when using unifipoller</summary>

```
# curl -s https://golift.io/repo.sh | sudo bash -s - unifi-poller
OK
Hit:1 http://us.archive.ubuntu.com/ubuntu focal InRelease
Hit:2 http://us.archive.ubuntu.com/ubuntu focal-updates InRelease
Hit:3 https://repos.influxdata.com/ubuntu focal InRelease
Hit:4 http://us.archive.ubuntu.com/ubuntu focal-backports InRelease
Hit:5 https://packages.grafana.com/oss/deb stable InRelease
Hit:7 http://us.archive.ubuntu.com/ubuntu focal-security InRelease
Get:9 https://pkgs.tailscale.com/stable/ubuntu focal InRelease
Hit:6 https://packages.graylog2.org/repo/debian sidecar-stable InRelease
Hit:8 https://packagecloud.io/golift/pkgs/ubuntu focal InRelease
Hit:10 https://packagecloud.io/sensu/stable/ubuntu focal InRelease
Fetched 5,542 B in 1s (4,595 B/s)
Reading package lists... Done
Building dependency tree
Reading state information... Done
33 packages can be upgraded. Run 'apt list --upgradable' to see them.
Reading package lists... Done
Building dependency tree
Reading state information... Done
The following NEW packages will be installed:
  unifi-poller
0 upgraded, 1 newly installed, 0 to remove and 33 not upgraded.
Need to get 5,213 kB of archives.
After this operation, 5,402 kB of additional disk space will be used.
Get:1 https://packagecloud.io/golift/pkgs/ubuntu focal/main amd64 unifi-poller amd64 2.1.3-884 [5,213 kB]
Fetched 5,213 kB in 1s (7,814 kB/s)
Selecting previously unselected package unifi-poller.
(Reading database ... 163595 files and directories currently installed.)
Preparing to unpack .../unifi-poller_2.1.3-884_amd64.deb ...
Unpacking unifi-poller (2.1.3-884) ...
Setting up unifi-poller (2.1.3-884) ...
Created symlink /etc/systemd/system/multi-user.target.wants/unifi-poller.service → /lib/systemd/system/unifi-poller.service.
Processing triggers for man-db (2.9.1-1) ...
```

</details>

# Potential solution
As a potential solution, the docs should be updated to refer to the correct package name in the Packagecloud repos.

**Comment by C-Duv** [comment]
I can confirm I got the same issue ("E: Unable to locate package unpoller") on a fresh Debian 11 which I resolved by using the provided command (which uses `unifi-poller`).

**Comment by platinummonkey** [comment]
should be fixed by #438
